src/shell/Makefile.am

```diff
@@ -79,6 +79,8 @@ flux_shell_SOURCES = \
 	output/filehash.c \
```

Comment:
commit message:

delete two "ands" in this sentence?
> Split the KVS output functionality and into output/kvs.[ch] and
under a new `struct kvs_output` abstraction.